src/test/java/com/smalaca/messagesender/controller/UserControllerTest.java

```diff
@@ -26,41 +27,73 @@
     public void shouldCreateNewUser() throws Exception {
         mvc.perform(MockMvcRequestBuilders.get("/user/create?login=andrzej&email=email").accept(MediaType.APPLICATION_JSON))
                 .andExpect(MockMvcResultMatchers.status().isOk())
-                .andExpect(MockMvcResultMatchers.content().string(CoreMatchers.equalTo("User Created")));
+                .andExpect(MockMvcResultMatchers.content().string(CoreMatchers.containsString("User Created")));
```

Comment:
Dlaczego nie equalTo? Przecież to ma być dokładnie taki String, czy się mylę?

Reply:
Tak chciałem zrobić, ale zwracając w kontrolerze Response, zwracam {succes = true......, message =User created} a nie samego Stringa. Myślisz że lepiej byłoby zwracać samego Stringa zamiast klasy Response ?